google-youtube-upload.html

```diff
@@ -393,6 +422,30 @@
         value: Boolean('MediaRecorder' in window &&
                        'mediaDevices' in navigator &&
                        'getUserMedia' in navigator.mediaDevices)
+      },
+
+      /**
+       * The [MediaStream audio constraints](https://developer.mozilla.org/docs/Web/API/MediaStreamConstraints/audio).
+       */
+      audio: {
+        type: Object,
+        value: true
+      },
+
+      /**
+       * The [MediaStream video constraints](https://developer.mozilla.org/docs/Web/API/MediaStreamConstraints/video).
+       */
+      video: {
+        type: Object,
+        value: true
+      },
+
+      /**
+       * The media type.
+       */
+      mimeType: {
+        type: String,
+        value: ''
```

Comment:
@e111077 BTW, is Polymer property default value optional? Is using `value: ''` equivalent to removing it from the code? If yes, should I remove it?